index.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path');
+const express = require('express');
+const morgan = require('morgan');
+const bodyParser = require('body-parser');
+
+const app = express();
+
+app.set('view engine', 'ejs');
+app.use(morgan('dev'));
+app.use(express.static(path.join(__dirname, 'public')))
+app.use(bodyParser.urlencoded({extended: false}));
+
+app.get('/', (request, response) => {
+  response.redirect('./clucks');
+});
+
+const clucksRouter = require('./routes/clucks');
+app.use('/clucks', clucksRouter);
+
+var time = new Date();
```

Comment:
since we are using ES6, you should no longer use `var`. Instead, you should use either `let` or `const`. In this case, since `time` is never reassigned a value (i.e. later on in your code you do not have `time = somethingElse`, `const` should be used